dedibox-network/rpn/how-to/mount-rpn-san-linux.mdx

```diff
@@ -0,0 +1,116 @@
+    ```
+    nano /etc/iscsi/iscsid.conf
+    ```
+
+3. Add the following configuration lines to the configuration: 
+    ```
+    node.conn[0].timeo.noop_out_interval = 0
+    node.conn[0].timeo.noop_out_timeout = 0
+    node.session.timeo.replacement_timeout = 86400
+    ```
+
+    The purpose of this modification is to prevent your file system from becoming read-only if the link is interrupted for more than two minutes (by default) between your server and the storage server.
+
+4. Adjust the setting for automatic reconnection of targets
+    ```
+    node.startup = automatic
+    ````
+
+5. Change the following settings to benefit from the data integrity check, if available
+```
+node.conn[0].iscsi.HeaderDigest = CRC32C,None
+node.conn[0].iscsi.DataDigest = CRC32C,None
+```
+
+## How to discover and connect to targets
+
+1. Run the following command to obtain the list of available targets: 
+    ```
+    iscsiadm -m discovery -t sendtargets -p SAN_SERVER
+    ```
+
+    <Message type="note">
+
+    The `SAN_SERVER` address is displayed in your Dedibox console.
+
+    </Message>
+
+    The target name should look like the following example: 
+    ```
+    X.X.X.X:3260,1 iqn.2013-01.net.online:XXXXXXXXXXX
+    ```
+
+2. Connect to the target by typing the following command: 
+    ```
+    iscsiadm -m node -T iqn.2013-01.net.online:XXXXXXXXXXX --login
+    ```
+
+3. Type `dmesg` to see the kernel logs. You should see some lines like in the following example
+    ```
+    scsi25 : iSCSI Initiator over TCP/IP
+    scsi 25:0:0:0: Direct-Access     IET      VIRTUAL-DISK     0    PQ: 0 ANSI: 4
+    sd 25:0:0:0: Attached scsi generic sg4 type 0
+    sd 25:0:0:0: [sdd] 2147483648 512-byte logical blocks: (1.09 TB/1.00 TiB)
+    sd 25:0:0:0: [sdd] Write Protect is off
+    sd 25:0:0:0: [sdd] Mode Sense: 77 00 00 08
+    sd 25:0:0:0: [sdd] Write cache: disabled, read cache: enabled, doesn't support DPO or FUA
+    sdd: sdd1
+    sd 25:0:0:0: [sdd] Attached SCSI disk
+    ```
+    In the example above, the disk detected is `sdd`, available at `/dev/sdd`. 
+
+    <Message type="note"> 
+
+    The assigned device name may be different, depending on your configuration.
+
+    </Message>
+
+    You can now use your SAN disk as any other disk.
+
+## Important recommendations for use
+
+As seen in the exaple abouve, the disk is available as `/dev/sdd.` It is not guaranteed that at the next reboot the same drive letter will be used, especially if you add other targets, or if your hardware configuration changes. It is therefore recommended not to use the names `/dev/sdX` in your configuration files.
+
+### How to address devices when using LVM
+LVM automatically uses UUIDs to recognize the partitions it uses. No problem in this case.
+
+### How to address devices when using direct disk formatting
+
+<Message typem="note">
+
+This method is not recommended, because it does not allow you to evolve afterwards, by aggregating several RPN-SAN storage spaces.
```

Comment:
```suggestion
This method is not recommended, because it does not allow you to make changes afterwards, by aggregating several RPN-SAN storage spaces.
```